engine/pokemon/stats_screen.asm

```diff
@@ -60,11 +60,10 @@ StatsScreenInit_gotaddress:
 StatsScreenMain:
 	xor a
 	ld [wJumptableIndex], a
-	; stupid interns
 	ld [wcf64], a
 	ld a, [wcf64]
 	and %11111100
-	or 1
+	or 1 ; ???
```

Comment:
Place the question marks at the start of the block, before `ld [wcf64], a`. We know what this line is doing (setting some 2-bit value in the byte) but not why.